pageserver/src/tenant/timeline.rs

```diff
@@ -413,7 +413,11 @@ pub struct Timeline {
     /// Keep aux directory cache to avoid it's reconstruction on each update
     pub(crate) aux_files: tokio::sync::Mutex<AuxFilesState>,
 
+    /// Size estiamtor for aux file v2
```

Comment:
This is part of https://github.com/neondatabase/neon/pull/7623, and the way it works is different from other timeline metrics. This is an approximate size incremented/decremented when new files get added/removed, and is not persisted.

Reply:
So why not put in the timelinemetrics?

Reply:
Sounds reasonable and I can move it in another pull request

Reply:
I noticed later that it is in fact a field in TimelineMetrics which should just be accessed directly instead of cloning it. None of the other fields of TimelineMetrics should be cloned as fields of Timeline.